web-server/src/content/DoraMetrics/DoraCards/WeeklyDeliveryVolumeCard.tsx

```diff
@@ -55,6 +56,11 @@ export const WeeklyDeliveryVolumeCard = () => {
   const dateRangeLabel = useCurrentDateRangeLabel();
   const deploymentFrequencyProps = useAvgIntervalBasedDeploymentFrequency();
 
+  const deploymentFrequencyCount = useCountUp(
+    deploymentFrequencyProps.count || 0,
+    1500
```

Comment:
Same thing here. Magic numbers. Ref: The other comment.

Reply:
fixed